vm/loader/loader_defs/src/shim.rs

```diff
@@ -96,6 +96,8 @@ open_enum! {
         /// and usermode. Today, this is only used for SNP: VMSA, CPUID pages,
         /// and secrets pages.
         VTL2_RESERVED = 7,
+        /// Memory preserved during servicing.
+        VTL2_PRESERVED = 8,
```

Comment:
Can we rename this to _exactly_ what it is? In this case, it is VTL2_GPA_POOL memory or something of the like. I want to avoid general terms, because its too easy to lump things together.

Reply:
IE see how I described it in this change to come soon:
```
        /// This memory is used by VTL2 usermode as a persisted GPA page pool.
        /// This memory is part of VTL2's address space, not VTL0's. It is
        /// marked as reserved to the kernel.
        VTL2_GPA_POOL = 8,
```
https://github.com/chris-oo/openvmm/commit/b057a1cab8cf10f1aee958e32eb497ee0cf4d5fa#diff-cf9d10c48283f6f5cd8120cd0465809d49f34c1fba5aa2ff5a7213ade1460ad8R102

`VTL2_PRESERVED` is too general once we start persisting saved state/other stuff/etc.

Reply:
Will rename

Reply:
Renamed to @chris-oo proposed name and even copied the comments for easier integration later.